apps/shared/src/modules/project/hooks/useTaskEvents.ts

```
@@ -0,0 +1,1081 @@
+
+	/** The last status snapshot body at-or-before the position. */
+	const statusAt = useCallback((): Record<string, unknown> | null => {
+		const position = cursorTimeRef.current;
+		if (position === null) return null;
+		const statusIndex = statusIndexRef.current;
+		const idx = upperBound(statusIndex, position) - 1;
+		if (idx >= 0) return (statusIndex[idx].body ?? null) as Record<string, unknown> | null;
+		// Before the first delivered snapshot the session's state-at-anchor
+		// seed answers (it already folded keyframe + interior to the anchor).
+		return seedStatusRef.current;
+		// eslint-disable-next-line react-hooks/exhaustive-deps
```

ESLint

[error] Definition for rule 'react-hooks/exhaustive-deps' was not found.

(react-hooks/exhaustive-deps)